tests/integration/components/polaris-skeleton-page-test.js

```diff
@@ -96,7 +98,13 @@ module('Integration | Component | polaris skeleton page', function(hooks) {
 
     // undefined title -> renders large skeleton display text.
     assert
-      .dom(buildNestedSelector(headerSelector, titleSelector))
+      .dom(
+        buildNestedSelector(
+          headerSelector,
+          titleAndPrimarySelector,
+          titleSelector
+        )
```

Comment:
Any reason not to define this selector as a `const` at the top of the file since we seem to be reusing it?

Reply:
Nope we can do that 👯‍♂️